pkgs/development/python-modules/mne-python/default.nix

```diff
@@ -1,53 +1,62 @@
 { lib
 , buildPythonPackage
 , fetchFromGitHub
-, isPy27
 , numpy
 , scipy
 , pytestCheckHook
-, pytest-cov
 , pytest-timeout
 , h5py
 , matplotlib
 , nibabel
 , pandas
 , scikit-learn
+, decorator
+, jinja2
+, pooch
+, tqdm
 }:
 
 buildPythonPackage rec {
   pname = "mne-python";
-  version = "1.0.3";
-
-  disabled = isPy27;
```

Comment:
This should be updated as upstream is limiting the their support to [>=3.7](https://github.com/mne-tools/mne-python/blob/main/setup.py#L101).